integration-tests/scripts/baseline/main.py

```diff
@@ -46,22 +46,78 @@
 # in the empty document to be explicit.
 EMPTY_BASELINE_STRUCTURE = []
 
+# In which format we expect to find dates in the baseline files
+DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+def object_parser_hook(data):
+    result = data
+
+    for f in (datetime_parser, memory_parser):
+        result = f(result)
+
+    return result
+
+
+def memory_parser(data):
+    result = {}
+    multipliers = {
+        "GiB": 1024,
+        "MiB": 1,
+        "KiB": 1 / 1024,
+    }
+
+    for k, v in data.items():
+        if k in ('Mem') and isinstance(v, str) and len(v) > 3:
```

Comment:
I think this either needs to be `k in ('Mem',)` to make it a tuple, or perhaps simply `k == 'Mem'` since there's only one value

Reply:
Right, there was something about the comma for tuples.